scripts/deck.js

```diff
@@ -4,37 +4,28 @@
 */
 
 class Deck {
-
-    deck = [];
-    static suits = ['Hearts', 'Spades', 'Clubs', 'Diamonds'];
-    static values = ['Ace', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King'];
-
-    // Construct deck
-    constructor() {
-        this.shuffle()
-        
+  deck = [];
+  static suits = ['Hearts', 'Spades', 'Clubs', 'Diamonds'];
+  static values = ['Ace', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King'];
+
+  // Construct deck
+  constructor () {
+    this.shuffle ();
+  }
+
+  // Shuffle method
+  shuffle () {
+    for (let suit in Deck.suits) {
+      for (let value in Deck.values) {
+        this.deck.push (`${Deck.values[value]} of ${Deck.suits[suit]}`);
+      }
     }
+  }
 
-    // Shuffle method
-    shuffle() {
-        for (let suit in this.suits) {
```

Comment:
What is this?

Reply:
Oh, the old way we were accessing suits was incorrect, since our class had Static values.